### PR TITLE
fix(TestUtils): export TS definitions with package

### PR DESCRIPTION
**Pull request by erautenberg**
## Description

Match Test Utils exports to UI-Components

## Checklist

- [ ] all commented code has been removed
- [ ] any new console issues have been resolved
- [ ] code linter and formatter has been run
- [ ] test coverage meets repo requirements
- [ ] PR name matches the expected semantic-commit syntax

### CODE REVIEW

File: packages/@lightningjs/ui-components-test-utils/package.json [hunk 1]
```diff
@@ -8,11 +8,16 @@
...
+  "exports": {
+    ".": {
+      "import": "./dist/lightning-ui-components-test-utils.min.mjs",
+      "require": "./dist/lightning-ui-components-test-utils.min.cjs",
+      "types": "./dist/lightning-ui-components-test-utils.d.ts"
+    },
+    "./src/": "./src/"
+  },
+  "main": "./dist/lightning-ui-components-test-utils.min.mjs",
+  "types": "./dist/lightning-ui-components-test-utils.d.ts",
```
Comment (THoj13) [inline packages/@lightningjs/ui-components-test-utils/package.json:20]: I don't see a `dist` folder generated in the test-utils dir. I think we need to also add a build script.

Comment (erautenberg) [reply packages/@lightningjs/ui-components-test-utils/package.json:20]: Good call. I tried adding this to match ui-components, but after `yarn build` I still don't see a dist folder 🤔 
```json
  "scripts": {
    "build": "rm -rf ./dist && rollup --config ../../../rollup/rollup.config.js --name lightning-ui-components-test-utils"
  },

**Comment by erautenberg** [comment]
Closing in favor of #287

**Comment by erautenberg** [comment]
Closing in favor of https://github.com/rdkcentral/Lightning-UI-Components/pull/287

**Pull request closed without merging**